graphqlapi/src/main/java/io/stargate/graphql/web/resources/cqlfirst/GraphqlDmlResource.java

```diff
@@ -163,4 +188,19 @@ private GraphQL getGraphl(
       return graphql;
     }
   }
+
+  private boolean isAuthorized(String token, String keyspaceName, Map<String, String> headers) {
+    try {
+      AuthenticationSubject authenticationSubject =
+          authenticationService.validateToken(token, headers);
```

Comment:
Could we pass the `AuthenticationSubject` into subsequent validation calls on the same request (to avoid re-authentication)?

This is a nice to have, not required for this PR, I think.

Reply:
Yes, there is a bit of duplication, we can definitely clean that up in a follow-up PR. I'll create at ticket.